specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json

```diff
@@ -276,16 +256,62 @@
         }
       }
     },
-    "/providers/Microsoft.AgFoodPlatform/farmBeatsExtensionDefinitions": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AgFoodPlatform/locations/{locations}/operationResults/{operationResultsId}": {
       "get": {
-        "tags": [
-          "FarmBeatsExtensions"
+        "tags": ["OperationResults"],
+        "description": "Get operationResults for a FarmBeats resource.",
+        "operationId": "OperationResults_Get",
+        "produces": ["application/json"],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
```

Comment:
>v2 [](http://example.com/codeflow?start=69&length=2)

non-blocking, consider updates these to latest version

Reply:
yes definitely, that is in our ToDos.